C-Plus-Plus/README.md

```diff
@@ -107,6 +107,7 @@ _add list here_
 - [Longest common sub sequence](dp/longest_increasing_subsequence.cpp)
 - [Longest increasing subsequence](dp/longest_increasing_subsequence.cpp)
 - [String permutations](dp/string_permutations.cpp)
+- [Maximum Sum Rectangle](dp/max_sum_rectangle.cpp)
```

Comment:
@sukritishah15 Update this in alphabetical order.

Reply:
Done @iamrajiv 
Please re-review.
Thank you!